azkaban-sql/src/sql/create-all-sql-3.0.0.sql

```diff
@@ -0,0 +1,217 @@
+CREATE TABLE active_executing_flows (
+	exec_id INT,
+	host VARCHAR(255),
+	port INT,
+	update_time BIGINT,
```

Comment:
You need to merge update sql script as well. https://github.com/azkaban/azkaban/tree/master/azkaban-sql/src/sql/update* ?